### PR TITLE
Refactor DeliveryControllerITest with deliveryShortDto

### PR DESCRIPTION
**Pull request by NastyaBondarenko**

### CODE REVIEW

File: goods-partner-backend/src/test/resources/response/delivery-controller-with-delivery-short-dto.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "routeCount": 1
+  },
+  {
+    "id": "123e4567-e89b-12d3-a456-556642440001",
+    "deliveryDate": "2022-09-19",
+    "status": "DRAFT",
+    "orderCount": 0,
+    "routeCount": 0
+  }
+]
```
Comment (BogdanParashchak) [inline goods-partner-backend/src/test/resources/response/delivery-controller-with-delivery-short-dto.json:16]: try to rename
maybe short-deliveries.json

File: goods-partner-backend/src/test/resources/response/delivery-controller.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          "unitWeight": 12.00,
+          "totalProductWeight": 12.00
+        }
+      ],
+      "orderWeight": 12.00,
+      "frozen": false
+    }
+  ],
+  "carLoads": []
+}
```
Comment (BogdanParashchak) [inline goods-partner-backend/src/test/resources/response/delivery-controller.json:66]: is it really to be located in /resources/response?
try to rename (maybe short-delivery-dataset.json)

File: goods-partner-backend/src/main/java/com/goodspartner/mapper/DeliveryMapper.java [hunk 3]
```diff
@@ -28,4 +30,15 @@ public interface DeliveryMapper {
     Delivery deliveryDtoToDelivery(DeliveryDto deliveryDto);
 
     List<DeliveryDto> deliveriesToDeliveryDtos(List<Delivery> deliveries);
+
+    @Named("deliveryToDeliveryShortDto")
+    default DeliveryShortDto deliveryToDeliveryShortDto(Delivery delivery) {
```
Comment (Ishei5) [inline goods-partner-backend/src/main/java/com/goodspartner/mapper/DeliveryMapper.java:35]: Можно использовать обычный маппинг, только для перемапливания коллекции в ее размер использовать expression, например:
@Mapping(target = "routeCount", expression = "java(delivery.getRoutes() != null ? delivery.getRoutes().size() : 0)")

**Review by Rooman — APPROVED** [review]

File: goods-partner-backend/src/test/resources/response/delivery-controller-with-delivery-short-dto.json (hunk 1 above)
Comment (NastyaBondarenko) [reply goods-partner-backend/src/test/resources/response/delivery-controller-with-delivery-short-dto.json]: renamed

File: goods-partner-backend/src/test/resources/response/delivery-controller.json (hunk 2 above)
Comment (NastyaBondarenko) [reply goods-partner-backend/src/test/resources/response/delivery-controller.json]: moved to delivery dataset

File: goods-partner-backend/src/main/java/com/goodspartner/mapper/DeliveryMapper.java (hunk 3 above)
Comment (NastyaBondarenko) [reply goods-partner-backend/src/main/java/com/goodspartner/mapper/DeliveryMapper.java]: дякую, використала

**Review by Ishei5 — APPROVED** [review]

**Pull request merged**